IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -445,7 +447,9 @@ - (NSDictionary *)wipeQuery
         wipeQuery = @{(id)kSecClass : (id)kSecClassGenericPassword,
                        (id)kSecAttrGeneric : [s_wipeLibraryString dataUsingEncoding:NSUTF8StringEncoding],
                        (id)kSecAttrAccessGroup : self.keychainGroup,
-                       (id)kSecAttrAccount : @"TokenWipe"};
+                       (id)kSecAttrAccount : @"TokenWipe",
+                       (id)kSecAttrService : @"TokenWipe"
```

Comment:
Seems that wipe query will not be able to get wipe data from ADAL keychain now because ADAL keychain doesn't store 'kSecAttrService'. Why do we need to store service here?

Reply:
For `kSecClassGenericPassword`, the combination of `kSecAttrAccount` and `kSecAttrService` forms the primary key for keychain. So it is safer to set both value.

The issue you point our absolutely legit, I made the change to set both values in write query and set only Account for read query :)